ext/openssl/ossl_pkey.c

```diff
@@ -1666,6 +1666,23 @@ ossl_pkey_decrypt(int argc, VALUE *argv, VALUE self)
     return str;
 }
 
+
+/*
+ * Returns bit length of the PKEy
```

Comment:
Please write rdoc comment in the similar format as other existing methods.

Also please include a reference to the relevant manpages.